### PR TITLE
Node.js 6.10 and base64 decode requests fix

### LINKED ISSUE #64: Why is application/json listed as a binary media type in example project?
Hi!
I've just spent a few hours troubleshooting a strange error I got only when deploying to AWS Lambda / API Gateway, and it turns out that it is because application/json is listed in the binaryMimeTypes collection in your example project. When I removed it, the POST and PATCH calls work just fine.

What is the reason for this, in what scenario are you interested in treating JSON input data as base64?

### PR DESCRIPTION
**Pull request by brettstack**
fix: decodes base64 requests (fixes #64); added form to index view to showcase POST/PUT;  move example instructions to example/README.md; update example to Node.js 6.10; added app.local.js to easily run local express

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -21,6 +21,15 @@ function getPathWithQueryStringParams(event) {
   return url.format({ pathname: event.path, query: event.queryStringParameters })
 }
 
+function getContentType(params) {
+  // only compare mime type; ignore encoding part
+  return params.contentTypeHeader ? params.contentTypeHeader.split(';')[0] : ''
+}
+
+function isContentTypeBinaryMimeType(params) {
```
Comment (kahouieong) [inline index.js]: Should we support wildcard here since API Gateway supports that

Comment (kahouieong) [inline index.js]: Should we support wildcard here since API Gateway supports that

Comment (brettstack) [inline index.js]: Do you mean major version bump? I considered this, but this is a bug fix that we want applied to version 2. However, there is a possibility to break anyone who fixed this on their Express side (e.g. via middleware).

Regardless of which we choose, the version bump will be a separate commit.

Comment (brettstack) [inline index.js]: Good point, but let's leave that as a separate issue #70

**Pull request merged**